### ISSUE TITLE
Error "Name 'flask' is not defined" when changing the force_log_out_after

### ISSUE THREAD

**Issue by matheusvasconcellos** [apache/airflow#10053]
**Apache Airflow version**: 1.10

**Kubernetes version (if you are using kubernetes)** (use `kubectl version`):

**Environment**:

- **Cloud provider or hardware configuration**: AWS
- **OS** (e.g. from /etc/os-release):
NAME="Amazon Linux"
VERSION="2"
ID="amzn"
ID_LIKE="centos rhel fedora"
VERSION_ID="2"
PRETTY_NAME="Amazon Linux 2"
- **Kernel** (e.g. `uname -a`): 4.14.186-146.268.amzn2.x86_64 (Amazon Linux 2)
- **Install tools**: Pip
- **Others**:

**What happened**:
When setting the **force_log_out_after** variable to a number different from the default 0, the error below is displayed when trying to login.

Other parameters that may be important:
**authenticate** = True
**auth_backend** = airflow.contrib.auth.backends.ldap_auth

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1947, in full_dispatch_request
    rv = self.preprocess_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2241, in preprocess_request
    rv = func()
  File "/usr/local/lib/python3.7/site-packages/airflow/www/app.py", line 198, in before_request
    flask.session.permanent = True
NameError: name 'flask' is not defined
```

**What you expected to happen**:
The user session should be forced to log out after the set amount of time.

**How to reproduce it**:
Change the **force_log_out_after** from the default value.

**Anything else we need to know**:

**Comment by ozw1z5rd** [comment]
There is at least another problem in the same block of code : 
`app.permanent_session_lifetime = datetime.timedelta(minutes=_force_log_out_after)`
but there is not import for `datetime` in  `airflow/www_rbac/app.py`, it imports `timedelta` so the correct code should be:
`app.permanent_session_lifetime = timedelta(minutes=_force_log_out_after)`

**Comment by ashb** [comment]
Fixed in 1.10.12
